keyboards/monsgeek/m3/info.json

```
@@ -0,0 +1,248 @@
+{
+    "keyboard_name": "M3",
+    "manufacturer": "MonsGeek",
+    "maintainer": "jonylee@hfd",
+    "usb": {
+        "vid": "0xFFFE",
+        "pid": "0x0009",
+        "device_version": "1.0.5",
+        "suspend_wakeup_delay": 400,
+        "force_nkro": true
+    },
+    "processor": "WB32FQ95",
+    "bootloader": "wb32-dfu",
+    "features": {
+        "bootmagic": true,
+        "mousekey": false,
+        "extrakey": true,
+        "console": false,
+        "command": false,
+        "nkro": true,
+        "encoder": false,
+        "rgb_matrix": true
+    },
```

migrated from `config.h`
```suggestion
    },
    "dynamic_keymap": {
        "layer_count": 6
    },
```